### ISSUE TITLE
Docker ID cannot reset account @ https://id.docker.com/reset-password

### ISSUE THREAD

**Issue by jdelreal** [docker/hub-feedback#1852]
Cannot reset Docker ID account via https://id.docker.com/reset-password
No password reset email ever arrives to associated email address inbox or spam/etc folders.

Have opened up two support tickets that has gone without responses [except automated acknowledgements of receipt], the first via directly emailing: 'Support@docker.com' and then weeks later by using https://success.docker.com/support. 

Would really appreciate any help, even if it was just a response from a real human, saying someone was actually looking into this issue, rather than simply sitting in a queue for weeks without being looked at even once.

TL;DR Could someone help me with my Docker ID issue, please. References:

Docker Support Ticket 00088332
Docker Support Ticket 00091365

Thanks,

-JDelReal

**Comment by invisibleU42** [comment]
I know that feel @jdelreal - I have the the same issue, that the support doesn't response.

I forgot the email address and the password of my Docker ID. In Docker's Terms of Service you can read in paragraph 4.3: "Accounts that are inactive for more than six months may be terminated at our discretion and without further notice". Therefore, I waited more than six months to be sure that my account is inactivated. Then I opened a Customer Service Request to claim my inactive Docker ID, as suggested in their article: https://success.docker.com/article/how-do-i-claim-an-inactive-docker-id

Except automated acknowledgement of receipt, I got no response. Two months later, I replied to the acknowledgement of receipt to ask for the current status - no response. 

Another two months later, I opened again a Customer Service Request as described above. And again, except automated acknowledgement of receipt, I got no response. And also again, two months later, I replied to the acknowledgement of receipt to ask for the current status - no response. 

TL;DR: Could someone help me with my Docker ID issue, please, references:

Docker Support Ticket 00067606
Docker Support Ticket 00071633

Thanks in advance!

**Comment by invisibleU42** [comment]
I know that feel @jdelreal - I have the the same issue, that the support doesn't response.

I forgot the email address and the password of my Docker ID. In Docker's Terms of Service you can read in paragraph 4.3: "Accounts that are inactive for more than six months may be terminated at our discretion and without further notice". Therefore, I waited more than six months in the hope, that my account get terminated due to inactivity, so that I can register it again, but it got not terminated and still exist. Then I opened a Customer Service Request to claim my inactive Docker ID, as suggested in their article: [https://success.docker.com/article/how-do-i-claim-an-inactive-docker-id](https://success.docker.com/article/how-do-i-claim-an-inactive-docker-id). Except automated acknowledgement of receipt, I got no response.

Two months later, I replied to the acknowledgement of receipt to ask for the current status - no response.

Another two months later, I opened again a Customer Service Request as described above. And again, except automated acknowledgement of receipt, I got no response. And also again, two months later, I replied to the acknowledgement of receipt to ask for the current status - no response.

TL;DR: Could someone help me with my Docker ID issue, please, references:

Docker Support Ticket 00067606
Docker Support Ticket 00071633

Thanks in advance!

**Comment by invisibleU42** [comment]
I know that feel @jdelreal - I have the the same issue, that the support doesn't response.

I forgot the email address and the password of my Docker ID. In Docker's Terms of Service you can read in paragraph 4.3: "Accounts that are inactive for more than six months may be terminated at our discretion and without further notice". Therefore, I waited more than six months in the hope, that my account get terminated due to inactivity, so that I can register it again, but it got not terminated and still exist. Then I opened a Customer Service Request to claim my inactive Docker ID, as suggested in their article: https://success.docker.com/article/how-do-i-claim-an-inactive-docker-id

Except automated acknowledgement of receipt, I got no response. Two months later, I replied to the acknowledgement of receipt to ask for the current status - no response.

Another two months later, I opened again a Customer Service Request as described above. And again, except automated acknowledgement of receipt, I got no response. And also again, two months later, I replied to the acknowledgement of receipt to ask for the current status - no response.

TL;DR: Could someone help me with my Docker ID issue, please, references:

Docker Support Ticket 00067606
Docker Support Ticket 00071633

Thanks in advance!

**Comment by jdelreal** [comment]
I'm still waiting on some help Docker Hub Support. No Reply here or via Email. 😢 👎

**Comment by jdelreal** [comment]
> bump
